servconf.c

```diff
@@ -1871,38 +1871,9 @@ process_server_config_line_depth(ServerOptions *options, char *line,
 		/* XXX appends to list; doesn't respect first-match-wins */
  parse_allowdenygroups:
 		while ((arg = argv_next(&ac, &av)) != NULL) {
-#ifdef WINDOWS
-			// it can be a SID string; if it is - use localized name for that SID
-			PSID Sid = NULL;
-			char* utf8_group_name = NULL;
-			wchar_t* arg_utf16 = utf8_to_utf16(arg);
-			if (ConvertStringSidToSidW(arg_utf16, &Sid) != 0) {
-				WCHAR group_name[UNLEN + 1];
-				DWORD group_name_length = UNLEN + 1;
-				WCHAR domain_name[DNLEN + 1] = L"";
-				DWORD domain_name_size = DNLEN + 1;
-				SID_NAME_USE account_type = 0;
-				if (LookupAccountSidW(NULL, Sid, group_name, &group_name_length,
-					domain_name, &domain_name_size, &account_type) != 0) {
-					utf8_group_name = utf16_to_utf8(group_name);
-					debug3_f("'%s' is translated to '%s'", arg, utf8_group_name);
-					arg = utf8_group_name;
-				} else {
-					debug3_f("LookupAccountSid failed for '%s'", arg);
-				}
-
-				if (Sid)
-					LocalFree(Sid);
-			}
-			else
-			{
-				debug3_f("'%s' not recognized as SID", arg);
-			}
-#endif // WINDOWS
-			if (*arg == '\0' ||
-				match_user(NULL, NULL, NULL, arg) == -1)
-					fatal("%s line %d: empty %s pattern",
-					filename, linenum, keyword);
+			if (*arg == '\0')
```

Comment:
What is this change for?

Reply:
this fixes the duplicate issue from merging - the `#ifdef WINDOWS` code is still defined below